src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -27,6 +26,12 @@ public DataProcessor(List<RoadAccident> roadAccidentList){
      * @return
      */
     public RoadAccident getAccidentByIndex7(String index){
+        Iterator<RoadAccident> iter = roadAccidentList.iterator();
+        while(iter.hasNext()) {
+            RoadAccident rec = iter.next();
```

Comment:
what is rec? :) 

There's a general recommendation to use some meaningful names for variables. with moder IDE it does not take too long time to press Ctrl+Space and get name like roadAccident. And it will save time to new-comers on a project when they read your code.